ephyviewer/traceviewer.py

```diff
@@ -574,6 +575,11 @@ def on_data_ready(self, t,   t_start, t_stop, visibles, dict_curves, times_curve
             return
 
         self.last_sigs_chunk = sigs_chunk
+        if sigs_chunk.shape[0] == 0:
+            for c in range(self.source.nb_channel):
+                self.curves[c].hide()
+            return
```

Comment:
In my tests I'm finding that returning early from ``on_data_ready`` prevents other things like the background color from being set properly, since that happens later in the function. I'd like to fix this before merging.

Reply:
OK I understand.
I can try to fix this as well.